lib/auth/sendVerificationRequest.ts

```diff
@@ -45,9 +45,86 @@ const html = (config: EmailConfig) => {
   const escapedEmail = `${email.replace(/\./g, '&#8203;.')}`;
 
   // HTML email template
+  // TODO: Replace logo URL with production URL
   return `
-        <p>Log into RCD APP platform as ${escapedEmail} by clicking the link below:</p>
-        <a href=${url}>Log into RCD</a>
+    <body
+      style="
+        background-color: #ffffff;
+        text-align: center;
+        font-family: Verdana, Arial', sans-serif;
+      "
+    >
+      <div
+        style="
+          height: 8px;
+          width: 100%;
+          background-color: #0B64CC;
+        "
+      />
+      <div
+        style="
+          padding: 32px 32px 40px;
+        "
+      >
+        <img
+          src="https://dev.rcdrichmondapp.ca/assets/logo.png"
+          style="
+            height: 65px;
+            width: 59px;
+            margin: 0 0 20px;
+          "
+          alt="Richmond Centre for Disability logo"
+        />
+        <div>
+          <h1
+            style="
+              color: #1A1A1A;
+              font-weight: bold;
+              font-size: 24px;
+              line-height: 150%;
+              margin: 0 0 20px;
+            "
+          >
+            Login
+          </h1>
+          <p
+            style="
+              font-size: 18px;
+              line-height: 150%;
+              margin: 0 0 32px;
+            "
+          >
+            Click the button below to authenticate as ${escapedEmail}
+            <br />
+            and sign into RCD APP Management System.
```

Comment:
just thinking out loud but it might be handy to make the "Platform name" a constant somewhere just so if we or RCD ever want it changed we don't have to go around looking everywhere

Reply:
Good point. However, the ideal solution is to confirm the platform name early so that we don't change it at all. It shouldn't be something that changes often, and the name of the platform isn't displayed a lot on the platform itself, so it might be a little overkill for our case. I will confirm the internal platform name with Carelynn and RCD ASAP.

Reply:
Updated the name!